docs/interpreter/bigquery.md

```diff
@@ -119,6 +124,28 @@ LIMIT
   40
 ```
 
+## Object Interpolation
+The BigQuery interpreter also supports interpolation of `ZeppelinContext` objects into the paragraph text.
+The following example shows one use of this facility:
+
+####In Scala cell:
+```
+z.put("country_code", "KR")
+    // ...
+```
+
+
+```bash
+%bigquery.sql
+    select * from patents_list where 
+    priority_country = '{country_code}' and filing_date like '2015-%'
+```
+
+Object interpolation is disabled by default, and can be enabled  by 
+setting the value of the property `zeppelin.bigquery.interpolation` to `true` (see _More Properties_ above). 
+More details of this feature can be found in the Spark interpreter documentation under 
+[Object Interpolation](spark.html#object-interpolation)
```

Comment:
is this the right link? it's been changed in other cases
https://github.com/apache/zeppelin/commit/941ccbbbdb6c2ca11f09f54bf0ddd31d6ba25d1c